Egeria-Operations.md

```diff
@@ -154,16 +154,77 @@ In between official releases, the latest build is also available to developers,
 
 ### Release Process Overview
 
-New releases can be created by Egeria maintainers that have the appropriate access
-on [Maven Central](https://oss.sonatype.org) and are set up to sign the release with gpg2.
+Releases are published to [Bintray](https://bintray.com/odpi) where they
+are GPG signed and distributed to [Maven
+Central](https://oss.sonatype.org).
 
 Creating a new release has the following stages:
 
-* Creating a branch of master for the release code.
-* Within the branch removing "-SNAPSHOT" from all of the Egeria version numbers in the pom files and committing with a pull request.
-* Pushing the release to [Maven Central](https://oss.sonatype.org).
-* Documenting the release with a [Git Release](https://github.com/odpi/egeria/releases).
+* Creating a branch off master for the release code.
 * Incrementing the release numbers in the pom files in master and committing through a pull request.
+* Within the branch removing "`-SNAPSHOT`" from all of the Egeria version
+  numbers in the pom files and committing with a pull request.
+* Staging a release through [Azure Pipelines](https://dev.azure.com/ODPi/Egeria/_release?_a=releases&definitionId=1&view=mine)
+* Verifying release criteria are met on the staged build
```

Comment:
Our team should expand on some of these other aspects - testing, licensing, CVEs, release notes etc - but in a separate PR.